### PR TITLE
edited to fix checkstyle / findbugs errors

### PR DESCRIPTION
**Pull request by dmalarev**

### CODE REVIEW

File: src/test/java/nl/tudelft/pixelperfect/SpaceshipTest.java [hunk 1]
```diff
@@ -29,6 +29,7 @@
  * @author Jesse Tilro
  *
  */
+@SuppressWarnings("PMD")
```
Comment (dalderliesten) [inline src/test/java/nl/tudelft/pixelperfect/SpaceshipTest.java]: Guess this will have to do for now.

**Comment by dalderliesten** [comment]
Looks fine to me.

File: src/test/java/nl/tudelft/pixelperfect/SpaceshipTest.java (hunk 1 above)
Comment (dmalarev) [inline src/test/java/nl/tudelft/pixelperfect/SpaceshipTest.java]: yeah, i fixed one of them, but the rest say the class has too many methods & static imports

Comment (dalderliesten) [inline src/test/java/nl/tudelft/pixelperfect/SpaceshipTest.java]: Fair enough, we'll look at it next Sprint.

**Comment by jessetilro** [comment]
LGTM

**Pull request merged**